style.css

```diff
@@ -0,0 +1,1427 @@
+  font-size: 14px;
+  padding: 10px 5px 10px 5px;
+}
+
+.last-line {
+  width: 100%;
+  background-color: rgb(0, 0, 0);
+  text-align: center;
+  height: 45%;
+}
+
+@media screen and (max-width: 1024px) and (min-width: 768px) {
+  nav {
+    display: flex;
+  }
+
+  .nav-container {
+    display: flex;
+    flex-direction: column;
+    width: 90vw;
+  }
+
+  .container-768 {
+    width: 10vw;
+    display: block;
+  }
+
+  .logo-for-768-logo {
+    background: black;
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+  }
+
+  .top-bar {
+    width: 90vw;
+    height: auto;
+    right: 0;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgb(0, 0, 0);
+  }
+
+  .top-left-bar a,
+  .top-right-bar a {
+    text-decoration: none;
+    color: rgb(172, 172, 172);
+    font-size: 0.7em;
+  }
+
+  .top-left-bar a {
+    padding-left: 3px;
+  }
+
+  .top-right-bar a {
+    padding-right: 3px;
+  }
+
+  .top-second {
+    width: 90vw;
+    height: auto;
+    display: flex;
+    right: 0;
+    justify-content: space-between;
+    padding: 10px 0 10px 0;
+    background-color: rgb(253, 253, 253);
+  }
+
+  .top-second-left a,
+  .top-second-right a {
+    text-decoration: none;
+    color: rgb(131, 131, 131);
+    font-size: 0.75em;
+    font-weight: 200;
+  }
+
+  .top-second-left a {
+    padding-left: 15px;
+  }
+
+  .top-second-right a {
+    padding-right: 10px;
+    padding-left: 5px;
+    border-left: 1px solid rgb(131, 131, 131);
+  }
+
+  .mobile-top-nav {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .tnw-logo {
+    display: none;
+  }
+
+  .section-one {
+    font-size: 70%;
+    width: 98vw;
+  }
+
+  .first {
+    background-image:
+      linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 50%),
+      url("./images/Opal-796x417.jpg");
+  }
+
+  .ln-head {
+    display: flex;
+    width: 95vw;
+    height: auto;
+    margin: auto;
+    justify-content: flex-start;
+    padding-bottom: 5px;
+  }
+
+  .latest-news {
+    width: 95vw;
+    height: auto;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 30vw 30vw 30vw;
+    grid-template-rows: 1fr 1fr 1fr;
+    margin-bottom: 8vh;
+  }
+
+  .news-img-1,
+  .news-img-2,
+  .news-img-3,
+  .news-img-4,
+  .news-img-5,
+  .news-img-6,
+  .news-img-7,
+  .news-img-8 {
+    display: flex;
+    flex-direction: column;
+    font-size: 85%;
+    padding-bottom: 10px;
+  }
+
+  .news-img-1 {
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  .news-img-2 {
+    grid-area: 1 / 2 / 2 / 3;
+  }
+
+  .news-img-3 {
+    grid-area: 1 / 3 / 2 / 4;
+  }
+
+  .news-img-4 {
+    grid-area: 2 / 1 / 3 / 2;
+  }
+
+  .news-img-5 {
+    grid-area: 2 / 2 / 3 / 3;
+  }
+
+  .news-img-6 {
+    grid-area: 2 / 3 / 3 / 4;
+  }
+
+  .news-img-7 {
+    grid-area: 3 / 1 / 4 / 2;
+  }
+
+  .news-img-8 {
+    grid-area: 3 / 2 / 4 / 3;
+  }
+
+  .section-heading {
+    font-size: 85%;
+  }
+
+  .info-card-768 {
+    display: none;
+  }
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#920` & `#1004` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?